EveHQ.HQF/Classes/Fitting.vb

```diff
@@ -1900,6 +1902,25 @@ Imports EveHQ.Common.Extensions
             End If
         Next
     End Sub
+
+
+	Private Sub ApplyResistanceKillerEffectsToShip(newShip As Ship)

```

Comment:
I already did something on line 1767, considering all absolute min and max should be processed last.
You may want to take a look and clear it up, from any side